### DISCUSSION TITLE
Way to search/replace a default chart value

### DISCUSSION THREAD

**Discussion by HasseJohansen** [fluxcd/flux2#4626] [Q&A] [answered]
Hi

I am using a HelmRelease to install Kyverno

Kyverno by default defines resourceFilters in their values.yaml like this:
```
resourceFilters:
  - '[Event,*,*]'
  - ...
  - '[Node,*,*]'
  - '[Node/*,*,*]'
  - ...
```

They are then taking these values and setting them in a configMap with a resourceFilters value like this:
```
resourceFilters: '[Event,*,*] ... [Node,*,*] [Node/*,*,*] ...'

So they basically stuff it in a string

I can override the whole resourceFilters helm value and remove the 2 Node entries I want to remove, but as it is quite a long list(and I guess it could be updated from time to time) I was wondering if there was a way to patch it?

As I understand right now I can only do postRenderers with strategicMerge and jsonpatch6902 which doesn't have operations for search/replace in a string. So I guess it is not possible?

Does anyone see a solution where I don't have to specify the whole list to get rid of the Node entries?

BR
Hasse

**Comment by HasseJohansen** [comment, marked as answer]
Ok. I found out that the Kyverno chart took a value named resourceFiltersExclude where I could state which entries to subtract from the default resourceFilters

It would be interesting though if there is an option to replace something in a default value string via a patch

**Reply by stefanprodan** [reply]
The JSON path spec has no such as concept of replacing a string in a value, you can only replace a JSON node.
